api/src/main/java/org/jboss/forge/roaster/model/util/Types.java

```diff
@@ -268,6 +268,11 @@ public static boolean isGeneric(final String type)
 
    public static String stripGenerics(final String type)
    {
+       return fixArray( type, true );
+   }
+
+   public static String fixArray(final String type, boolean stripGenerics)
```

Comment:
The original method did more than what its name - "stripGenerics" - implied. I wonder if some more refactoring is needed?